client/hosting/performance/components/PerformanceReport.tsx

```diff
@@ -28,6 +31,8 @@ export const PerformanceReport = ( {
 	filter,
 	onFilterChange,
 }: PerformanceReportProps ) => {
+	const isLoggedIn = useSelector( isUserLoggedIn );
```

Comment:
This temporarily adds the redux selector to `/hosting`, however this package will ultimately be retired as we shift to the Hosting Dashboard V2

Reply:
This is necessary to make sure it still works on the existing hosting dashboard.